mesop/web/src/component_renderer/component_renderer.ts

```diff
@@ -221,8 +221,9 @@ export class ComponentRenderer {
   }
 
   private updateCustomElementChildren() {
-    const existingChildren = Array.from(
-      this.customElement!.querySelectorAll(COMPONENT_RENDERER_ELEMENT_NAME),
+    const existingChildren = Array.from(this.customElement!.children).filter(
+      (child) =>
+        child.tagName === COMPONENT_RENDERER_ELEMENT_NAME.toUpperCase(),
```

Comment:
Maybe add a comment here on why we need to do toUpperCase here. It's not immediately clear to me what's going on. I see the constant is `component-renderer-element` and when we create the element we use `component-renderer-element`. So why does the tag name change to `COMPONENT-RENDERER-ELEMENT`?

Uh, ok, I looked it up. I see. So tagName returns all uppercase so it can be case-insensitive.

Reply:
yup - added a comment